### DISCUSSION TITLE
Auto3d doesn't set OMP_NUM_THREADS

### DISCUSSION THREAD

**Discussion by AHarouni** [Project-MONAI/MONAI#5753] [Ideas]
**Describe the bug**
Running Auto3d I get warning of 
```
OMP_NUM_THREADS set to 1 to avoid system overload
```

we should smartly export this variable before the torchrun 

I tired exporting it myself but this doesn't work with multiple nodes. I think this parameter needs to be set in the python code itself

**Comment by wyli** [comment]
I think it is nice that torchrun automatically modifies the system setting for better performance and creates a warning message to notify the user. what's not working with multi-node?

**Reply by myron** [reply]
@AHarouni it's one of the pytorch/torchrun info messages.  we've all been seeing it a lot even in other codes.  I don't think we should set it (to keep it auto), but perhaps we can hide that warning somehow (or just leave it be)
